README.md

```diff
@@ -19,141 +20,189 @@ When the reader has completed this Code Pattern, they will understand how to:
+  * Click `Save`.
 
-* Move your cursor to each code cell and run the code in it. Read the comments for each cell to understand what the code is doing. **Important** when the code in a cell is still running, the label to the left changes to **In [\*]**:.
-  Do **not** continue to the next cell until the code is finished running.
+  ![config_map.png](doc/source/images/config_map.png)
 
-# Sample output
+* Go to the `Applications` tab, choose `Deployments` to view the status of your application.
 
-### GET the application server deployments
+## 3. Get a secure endpoint
 
-Navigate a browser to `http://<ip_address>:<port>/v1/deployments`
-or run the [test_deployments.py](test/deployments_api.py) script following the  [test/README.md](test/README.md) instructions.
+* From the OpenShift or OKD UI, under `Applications` ▷ `Routes` you will see your app.
+  * Click on the application `Name`.
+  * Under `TLS Settings`, click on `Edit`.
+  * Under `Security`, check the box for `Secure route`.
+  * Hit `Save`.
 
+## 4. Run the web app
 
-Output:
+* Go back to `Applications` ▷ `Routes`. You will see your app.
 
-```
-{"count":3,"resources":[{"entity":{"asset":{"guid":"resnet50","name":"resnet50"},"asset_properties":{"input_data_type":"unstructured_image","problem_type":"multiclass"},"description":"Keras ResNet50 model deployment for image classification","name":"ResNet50 AIOS compliant deployment","scoring_url":"http://169.60.16.73:31520/v1/deployments/resnet50/online"},"metadata":{"created_at":"2016-12-01T10:11:12Z","guid":"resnet50","modified_at":"2016-12-02T12:00:22Z"}},{"entity":{"asset":{"guid":"resnet50","name":"resnet50"},"asset_properties":{"input_data_type":"unstructured_image","problem_type":"multiclass"},"description":"Keras ResNet50 model deployment for image classification","name":"ResNet50 AIOS non compliant deployment","scoring_url":"http://169.60.16.73:31520/v1/deployments/resnet50_non_compliant/online"},"metadata":{"created_at":"2016-12-01T10:11:12Z","guid":"resnet50_non_compliant","modified_at":"2016-12-02T12:00:22Z"}},{"entity":{"asset":{"guid":"action","name":"area and action prediction"},"asset_properties":{"input_data_type":"structured","problem_type":"multiclass"},"description":"area and action spark models deployment","name":"action deployment","scoring_url":"http://169.60.16.73:31520/v1/deployments/action/online"},"metadata":{"created_at":"2016-12-01T10:11:12Z","guid":"action","modified_at":"2016-12-02T12:00:22Z"}}]}
-```
+* Save the URL for the Route. You will use this in the [configuration notebook](#5-create-a-notebook-on-cloud-pak-for-data) in the section `1.2 Configure credentials` for the variable *CUSTOM_ML_PROVIDER_SCORING_URL*.
 
-### Run the [score_credit.py](test/score_credit.py) script following the  [test/README.md](test/README.md) instructions
+[![return](https://raw.githubusercontent.com/IBM/pattern-utils/master/deploy-buttons/return.png)](../../README.md#create-a-watson-openscale-service)
```

Comment:
Don't need this return button. You never left the README.md page

Reply:
fixed